react/modules/tasks/routing.md

```diff
@@ -71,5 +79,7 @@ A student can achieve a maximum of 100 points.
 - Presence of _code-smells_ (God-object, chunks of duplicate code), commented code sections: **-10 points per each**
 - Usage of Redux or other state management libraries: **-100 points**
 - Usage of component libraries, e.g. Material UI, Ant Design: **-100 points**
+- Test coverage <70% but >= 60% (at least for `statements`): **-10 points**
```

Comment:
I suggest to add a point:
- Test coverage below 60% (al least for `statements`): **-50 points**

Reply:
added